src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -36,13 +37,21 @@
                       12:01 AM (Pacific Time). No other filings are allowed until then.</span>
                   </v-tooltip>
                 </v-scale-transition>
+
+                <template v-else-if="isPaid(filing.status)">
+                  <span>FILING PENDING (filed by {{filing.filingAuthor}} on {{filing.filingDate}})</span>
```

Comment:
Is this a temporary text? As far as I know, the design for this is awaiting review

Reply:
I confirmed this with Scott.

Reply:
Scott has provided updated mockups (including an expansion detail) and I have now implemented that as well.